src/api/app/models/review.rb

```diff
@@ -94,23 +97,31 @@ def check_initial
       errors.add(:unknown, 'no reviewer defined')
     end
 
-    if by_user && !User.find_by_login(by_user)
+    if by_user
```

Comment:
So, assigning a variable (not used anymore) to true or nil (because is not set before) by having a flow control structure of 2 levels of conditions is more clear than just a composited condition? I don't see it... I can understand the "complex" condition without any problem... the other is just hurting my eyes 😈